.github/workflows/build.yaml

```diff
@@ -21,6 +21,11 @@ jobs:
           distribution: 'zulu'
           java-version: 11
 
+      # For watchOS x86 simulator. Remove with Kotlin 1.8.
+      - uses: maxim-lobanov/setup-xcode@v1
```

Comment:
Do you have a youtrack ticket? I got the same error with Xcode 14.

Reply:
It's already fixed on their `master` so I didn't bother. The file is called `ExecutorService.kt` in their repo.

Reply:
Unfortunately it looks like it's currently only in Kotlin 1.8.20 according to GitHub, but perhaps it was cherry-picked 🤞